src/Illuminate/Cache/DynamoDbStore.php

```diff
@@ -102,21 +102,13 @@ public function get($key)
             ],
         ]);
 
-        if (! isset($response['Item'])) {
+        if (! isset($response['Item']) || $this->isExpired($response['Item'])) {
             return;
         }
 
-        if ($this->isExpired($response['Item'])) {
-            return;
-        }
+        $value = $response['Item'][$this->valueAttribute]['S'] ?? $response['Item'][$this->valueAttribute]['N'] ?? null;
 
-        if (isset($response['Item'][$this->valueAttribute])) {
-            return $this->unserialize(
-                $response['Item'][$this->valueAttribute]['S'] ??
-                $response['Item'][$this->valueAttribute]['N'] ??
-                null
-            );
-        }
+        return isset($value) ? $this->unserialize($value) : null;
```

Comment:
Is checking `isset()` still correct? as it always fallback to `null`.

Reply:
I believe that `is_null()` or just plain old `=== null` be best in this situation.

Reply:
What's the different between `return $this->unserialize(null):` vs `return null;`?

Reply:
@crynobone how it always fallback to `null`? 

`return isset($value) ? $this->unserialize($value) : null;`

this checks if `$value` is set then unserialize it

Reply:
I tested it (copied it into tinker so maybe not really) and passing null into `unserialize` throws an error.

Reply:
@miladev95 Well, you just declared `$value` on the line above. So while `isset($value)` resolves to false if the value is null, it also seems a bit confusing when you see it assigned a value in the line before.